### ISSUE TITLE
Not able to see Betterttv emotes in some channels

### ISSUE THREAD

**Issue by mkhan77** [night/betterttv#5711]
Not able to see any betterttv emotes in some channels especially xqc. I have attached the javascript console as there is some error some the betterttv.js file. I have tried to reset cache and cookies. I even reinstalled chrome from scratch, but it still did not solve the issue. 
![betterttv issue](https://user-images.githubusercontent.com/26639431/198945282-a771814c-6b14-4861-9e3f-376fe44e6428.png)
Please advise as this has been happening for weeks.

**Comment by Rock1e** [comment]
yeah its been happening to me too. after i did a factory reset on my pc and re-downloaded the extension it has been buggin out too and not showing emotes

**Comment by night** [comment]
The extension is not "bugging out." The emotes in streams you do not see are not BetterTTV emotes. You can see BetterTTV emotes available to you in the BetterTTV emote menu or the BetterTTV website at `https://betterttv.com/channels/CHANNELNAME`

**Comment by mkhan77** [comment]
Actually, if you see in the screenshot, the "OMEGALUL" emote did not show up and that is a global betterttv emote. Would you know why that is?

**Comment by Flamerdragon** [comment]
the same is happening with me. Emotes like OMEGALUL are global emotes and they are not showing up for me.

**Comment by night** [comment]
`OMEGALUL` is not a global emote. You can see a list of global emotes at https://betterttv.com/emotes/global
